Library/Homebrew/formulary.rb

```diff
@@ -241,14 +243,34 @@ def self.core_path(name)
     Pathname.new("#{HOMEBREW_LIBRARY}/Formula/#{name.downcase}.rb")
   end
 
-  def self.tap_paths(name)
+  def self.tap_paths(name, taps=Dir["#{HOMEBREW_LIBRARY}/Taps/*/*"])
```

Comment:
Let's make it `Dir["#{HOMEBREW_LIBRARY}/Taps/*/*/"]` so it only matches directories.